code/modules/projectiles/guns/launcher/rocket.dm

```diff
@@ -38,8 +38,4 @@
 		rockets -= I
 		return new I.projectile_type(src)
 	return null
-
-/obj/item/weapon/gun/launcher/rocket/handle_post_fire(mob/user, atom/target)
```

Comment:
This should be left in (not the admin log/message, just the proc) - launchers may need to do a callback post-fire, but if you're set on removing it, remove the ..() below.